diff --git a/dist/cjs/index.js b/dist/cjs/index.js
index 2ccec58014850d344d8696b6cf5f7f8b76eb6dc5..df3789ec056cfc1b1c91006295350477ebcc9f7e 100644
--- a/dist/cjs/index.js
+++ b/dist/cjs/index.js
@@ -7,3 +7,18 @@ Object.defineProperty(exports, "imageSize", { enumerable: true, get: function ()
 Object.defineProperty(exports, "default", { enumerable: true, get: function () { return lookup_1.imageSize; } });
 var types_1 = require("./types");
 Object.defineProperty(exports, "types", { enumerable: true, get: function () { return types_1.types; } });
+
+var fs = require("fs");
+var path = require("path");
+
+function wrappedImageSize(input) {
+  if (typeof input === "string") {
+    input = fs.readFileSync(path.resolve(input));
+  }
+  return (0, lookup_1.imageSize)(input);
+}
+
+module.exports = wrappedImageSize;
+Object.assign(module.exports, exports);
+module.exports.default = wrappedImageSize;
+module.exports.imageSize = wrappedImageSize;
